jvb/src/main/kotlin/org/jitsi/videobridge/EndpointConnectionStatusMonitor.kt

```diff
@@ -129,8 +129,13 @@ class EndpointConnectionStatusMonitor @JvmOverloads constructor(
             // debugging purposes, and we also broadcast it through Octo.
             conference.broadcastMessage(msg, true)
         } else {
-            val ep = conference.getEndpoint(receiverEpId)
-            conference.sendMessage(msg, listOf(ep), false)
+            val ep = conference.getLocalEndpoint(receiverEpId)
+            if (ep != null) {
+                conference.sendMessage(msg, listOf(ep), false)
+            } else {
+                /* TODO: send only to the relevant relays */
+                conference.broadcastMessage(msg, true)
```

Comment:
I don't think this needs to be broadcast. The case where `receiverEpId` is not null is when its associated endpoint connected and needs to be notified of other endpoints' status. With just a bit of changes we can pass the `Endpoint` object itself instead its ID.

Reply:
Ah, there's never a case where we need to direct this to a remote endpoint, ok.

The main issue with changing this to take an `Endpoint` would be adjusting `EndpointConnectionStatusMonitorTest`, which uses a lot of mockk-code which I don't understand very well.